### PR TITLE
creates /api/reviews with a query for categories with tdd

### PR DESCRIPTION
**Pull request by YousuffTanveer**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -193,3 +193,55 @@ describe("PATCH /api/review/:review_id ", () => {
       });
   });
 });
+
+describe("status 200, GET /api/reviews", () => {
+  test("Status 200 return an array of reviews", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { category } }) => {
```
Comment (scar1377) [inline __tests__/app.test.js:202]: Maybe thinking about changing category to reviews to reduce the confusion

File: __tests__/app.test.js [hunk 2]
```diff
@@ -193,3 +193,55 @@ describe("PATCH /api/review/:review_id ", () => {
...
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+        expect(category.length).toBe(1);
+      });
+  });
```
Comment (scar1377) [inline __tests__/app.test.js:246]: Need some error handling when category does not exist

**Review by scar1377 — CHANGES_REQUESTED** [review]
Just need some sad path for category query

**Review by HeatherEver — COMMENTED** [review]
Nice work Yousuff, more tests could be added for some errors i.e non-existent id, invalid id

**Pull request merged**